internal/k8s/gvrautodetect.go

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2020 VMware
```

Comment:
Is the name `gvrautodetect.go` still appropriate? or should we rename it `informerset.go`, or simply `informers.go`?

Reply:
Yep, good idea.